add notes for n26158 (#569)
bitcoin-core-review-club/website: _posts/2022-10-12-#26158.md

1. What is the initial motivation behind this proposed pchange? Does the approach in this PR sufficiently address that? Can you think of another, not necessarily (strictly) better, approach?

1. Are there any downsides to `make check` only running `-priority-level=high` benchmarks by default? Do you think the benefits outweigh the downsides?

1. Why do we need to `static_cast<PriorityLevel>` the return value when [`operator|(PriorityLevel a, PriorityLevel b)`](https://github.com/bitcoin-core-review-club/bitcoin/blob/cbc077e77a5d0ba8ae11b1f5d89480c997fdef55/src/bench/bench.h#L50)'s return type is `PriorityLevel` already?

1. In which sense is `PriorityLevel` an enum? In which sense is it a bit vector? What do you think about this approach?

1. Which levels can we pass to `-priority-level`? Are they all represented in [`map_priority_level`](https://github.com/bitcoin-core-review-club/bitcoin/blob/cbc077e77a5d0ba8ae11b1f5d89480c997fdef55/src/bench/bench.cpp#L47-L51)? How are the other one(s) resolved? Can you think of a different approach?

> 1. Were you able to compile and run the benchmarks? How did you, or could you, test that the `-priority-level` argument works as expected?

Reviewer: ```suggestion
1. Were you able to compile and run the benchmarks? How did you test that the `-priority-level` argument works as expected?
```

1. Which benchmarks do you think should/could be labeled as `LOW` or `MEDIUM` priority? What did you base that decision on?

> 1.  Which of the below do you think are valid `PriorityLevel`s:

Reviewer: ```suggestion
1.  Quiz: Which of the following are valid `PriorityLevel`s:
```
